wrapper.js

```diff
@@ -22,17 +24,26 @@ exports.init = function(root, config) {
       var locals = {};
 
       // Merge any locals passed to config.locals
-      if (config.locals && typeof(config.locals) === 'object')
-        for (var attrname in config.locals) { locals[attrname] = config.locals[attrname]; }
+      if (config.locals && typeof(config.locals) === 'object') {
+        for (var attrname in config.locals) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.